code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -95,6 +95,14 @@
 
```

Comment:
don't forget to update aggressive spread's description

Reply:
the charm of late night edits...